bigtable/CMakeLists.txt

```diff
@@ -128,6 +128,8 @@ add_library(bigtable_data_api
         client/data.cc
         client/idempotent_mutation_policy.h
         client/idempotent_mutation_policy.cc
+        client/detail/bulk_mutator.h
```

Comment:
We can separately debate if we should call this `detail`, or `internal`, or `impl`, or something else at some point, and whether the headers should stay as-is, or if we should create a `public` subdirectory for headers (so that it's clear what can be used and what shouldn't).

This should probably be discussed in the coding style guide.

Reply:
Ack, and agree.